cve_bin_tool/scanner.py

```diff
@@ -12,6 +12,22 @@
 from .strings import Strings
 from .util import inpath, DirWalk
 
+from typing import NamedTuple, List
+
+
+class CVE(NamedTuple):
+    cve_id: str
+    severity: str
+    description: str = ""
+
+
+class CVEData(NamedTuple):
+    vendor: str
+    product: str
+    version: str
+    cves: List[CVE]
+    checker: str = ""
```

Comment:
Do we need it? Since you have mentioned we are using product for testing scanner now.

Reply:
I thought it might help you in the future to distinguish between products. But yeah you can add it later if you need. So I'm going to remove that.

Reply:
I just suggested it because test weren't passing but if it is working fine with newer system then remove it.